consensus/qbft/src/main/java/org/hyperledger/besu/consensus/qbft/statemachine/QbftRound.java

```diff
@@ -102,26 +100,28 @@ public void createAndSendProposalMessage(final long headerTimeStampSeconds) {
   }
 
   public void startRoundWith(
-      final RoundChangeMetadata roundChangeMetadata, final long headerTimestamp) {
-    final Optional<Block> bestBlockFromRoundChange = roundChangeMetadata.getBlock();
+      final RoundChangeArtifacts roundChangeArtifacts, final long headerTimestamp) {
+    final Optional<PreparedCertificate> peerWasPrepared =
+        roundChangeArtifacts.getBestPreparedPeer();
+
     Block blockToPublish;
-    if (bestBlockFromRoundChange.isEmpty()) {
+    if (peerWasPrepared.isEmpty()) {
```

Comment:
Don't like this variable name, it implies a boolean to me. Perhaps bestPreparedCertificate?